src/index.js

```diff
@@ -1,18 +1,14 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
-import { Provider } from 'react-redux';
 import { BrowserRouter as Router } from 'react-router-dom';
-import store from './redux/store';
 import App from './App';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
-    <Provider store={store}>
-      <Router>
-        <App />
-      </Router>
-    </Provider>
+    <Router>
+      <App />
+    </Router>
```

Comment:
- You have done great so far :+1: :+1: But there are some requirements you still need to meet before moving to the next miksetone. :+1: :+1: 

- - Screenshot reference:
![Screenshot from 2023-09-30 20-22-49](https://github.com/Kingno3l/Bookstore/assets/107632780/7674e72a-a67a-4fcf-8bc3-582d415fd498)

- [ ] As stated in the project requirement, you need to Wrap the App component inside the Provider component from redux and pass the store to the provider. This will allow you to have access to the redux state in every part of your components. :+1: :100: I'll suggest you read this thoroughly to understand it. :+1: 
EG:
```sh
import store from './redux/store';
import './index.css';
import App from './App';

const root = ReactDOM.createRoot(document.getElementById('root'));
root.render(
  <React.StrictMode>
    <Provider store={store}>
      <BrowserRouter>
        <App />
      </BrowserRouter>
    </Provider>
  </React.StrictMode>,
);
```

Reply:
- [ ] As requested by the previous reviewer, you need to Wrap the App component inside the Provider component from redux and pass the store to the provider. This will allow you to have access to the redux state in every part of your components. 👍 💯 I'll suggest you read this thoroughly to understand it. 👍
EG: